thompson_sampling.py

```diff
@@ -1,7 +1,7 @@
 import math
 import random
 import sys
```

Comment:
I don't know if this resulted from the changes in this PR but these two imports are unused.